src/pages/home/report/ReportActionCompose.js

```diff
@@ -704,6 +700,7 @@ class ReportActionCompose extends React.Component {
                     <ReportTypingIndicator reportID={this.props.reportID} />
                     <ExceededCommentLength commentLength={this.comment.length} />
                 </View>
+                {this.state.isDraggingOver && <ReportDropUI />}
```

Comment:
Now this can be taken out of `ReportactionComposer`. May be just put in the DragAndDropUI itself.

Reply:
This would require larger restructuring since we depend on displayFileInModal inside onDrop which is provided from AttachmentModal

Reply:
Then you can put the whole object in the ReportActionCompose component.